src/components/Dialogs/DialogManager.js

```diff
@@ -31,11 +33,13 @@ const DialogManager = () => {
     if (isDynamicDimension(dimension?.dimensionType)) {
         return <DynamicDimension dimension={dimension} onClose={onClose} />
     }
+    if (dimension?.dimensionType === DIMENSION_TYPE_PERIOD) {
```

Comment:
The logic in this file is ready for a cleanup. I think it should be possible to do a switch on `dimensionType`:  

```
    if (!dimension?.id) {
        return null
    }

    switch (dimension.dimensionType) {
        case DIMENSION_TYPE_CATEGORY:
        case DIMENSION_TYPE_CATEGORY_OPTION_GROUP_SET:
        case DIMENSION_TYPE_ORGANISATION_UNIT_GROUP_SET: {
            return <DynamicDimension dimension={dimension} onClose={onClose} />
        }
        case DIMENSION_TYPE_PERIOD: {
            return <PeriodDimension dimension={dimension} onClose={onClose} />
        }
        case DIMENSION_ID_ORGUNIT: {
            return <FixedDimension dimension={dimension} onClose={onClose} />
        }
        default: {
            return <ConditionsManager dimension={dimension} onClose={onClose} />
        }
    }
```

Also. the FixedDimension component should be cleaned up too. Looks like it was expecting to handle PERIOD in the future, but now it probably will only ever handle OU, since there is a separate PeriodDimension component. Right?

Reply:
@jenniferarnesen I think you are probably correct, but I wonder if this is something we should include in this PR, since we want to get this merged ASAP to unblock the create workflow in the new app...